cassandra/cluster.py

```diff
@@ -2124,14 +2124,14 @@ def prepare(self, query, custom_payload=None):
         future = ResponseFuture(self, message, query=None, timeout=self.default_timeout)
         try:
             future.send_request()
-            query_id, column_metadata, pk_indexes = future.result()
+            query_id, bind_metadata, pk_indexes, result_metadata = future.result()
```

Comment:
@aboudreault I created https://datastax-oss.atlassian.net/browse/PYTHON-601 to refactor at some point. We might be able to work it into this release if there is time.